pkg/entities/types.go

```diff
@@ -17345,6 +17345,7 @@ type EntityOutlineInterface interface {
 	GetGUID() common.EntityGUID
 	GetName() string
 	GetType() string
+	GetTags() []EntityTag
```

Comment:
for anyone finding this PR in future: no other changes are needed in this file, since Tutone seems to have already populated all fields implementing the `EntityOutlineInterface` with a Tags attribute and a `GetTags` method.

This has been verified and checked multiple times by @abasha1234 and me.